app/utils/error_handlers.py

```diff
@@ -0,0 +1,98 @@
+import typing as tp
+
+from flask import Response, jsonify
+from pydantic.error_wrappers import ValidationError
+from sqlalchemy.exc import IntegrityError
+
+from utils.exceptions import (
+    LimitOffsetError,
+    SortException,
+    PermissionException,
+    EmptyBodyException,
+    SerializerValidationError,
+)
+
+
+def validation_error_handler(error: ValidationError) -> tp.Tuple[Response, int]:
+    """
+    Except ValidationError
+    Args:
+        error: current error
+    Returns:
+        json with status code
+    """
+
+    return jsonify(error.errors()), 400
+
+
+def limit_offset_error_handler(error: LimitOffsetError) -> tp.Tuple[Response, int]:
+    """
+    Except LimitOffsetError
+    Args:
+        error: current error
+    Returns:
+        json with status code
+    """
+
+    return jsonify({'Error': error.message}), 400
+
+
+def sort_error_handler(error: SortException) -> tp.Tuple[Response, int]:
+    """
+    Except LimitOffsetError
+    Args:
+        error: current error
+    Returns:
+        json with status code
+    """
+
+    return jsonify({'Error': error.message}), 400
+
+
+def integrity_error_handler(error: IntegrityError) -> tp.Tuple[Response, int]:
+    """
+    Except IntegrityError
```

Comment:
you don't need to handle SQLAlchemy errors

Reply:
done